packages/nuxt/src/core/plugins/import-protection.ts

```diff
@@ -85,7 +88,7 @@ export const ImportProtectionPlugin = createUnplugin(function (options: ImportPr
         matched = true
       }
       if (matched) {
-        return _require.resolve('unenv/runtime/mock/proxy')
+        return proxy
```

Comment:
```suggestion
        return resolvePath('unenv/runtime/mock/proxy', { url: useNuxt().options.modulesDir })
```

Reply:
I thought it might be more performant to do this once to rather than every time the id is resolved...